log4j2-elasticsearch-core/src/main/java/org/appenders/log4j2/elasticsearch/VirtualProperty.java

```diff
@@ -62,6 +77,10 @@ public boolean isDynamic() {
         return dynamic;
     }
 
+    public boolean writeRaw() {
```

Comment:
Convention: I think it should be `isWriteRaw()` regardless of how it looks and sounds.